src/compas_viewer/viewer.py

```diff
@@ -247,17 +250,17 @@ def renderer():
 
     def add(
         self,
-        item: Union[Mesh, Geometry, "OCCBrep", "Graph"],
+        item: Union[Mesh, Geometry, "OCCBrep", "Graph", "RobotModel"],
```

Comment:
I think its ok to just say `item: Data` where `Data is compas.data.Data`. Because people will create custom classes from `Data` class and register their own custom `Objects`. They are not necessarily one of these